task.go

```diff
@@ -208,7 +208,7 @@ func (r *marathonClient) TaskEndpoints(name string, port int, healthCheck bool)
 	return list, nil
 }
 
-func (r *Task) allHealthChecksAlive() bool {
+func (r *Task) AllHealthChecksAlive() bool {
```

Comment:
@timoreimann , I prefer option 1 , leave it to people who use this method, as if health check differ from task to task, people who set it should know.